.github/workflows/ci.yml

```diff
@@ -47,7 +47,7 @@ jobs:
             coverage-name: oldest
             python: "3.7"
             numpy-version: numpy==1.16.0
-            tf-version: tensorflow==2.2.0
+            tf-version: tensorflow==2.3.4
```

Comment:
I'm not sure how it's normally done, but if we're claiming support with `2.3` we should probably test with 2.3.0 unless it's not available on PyPI?

Reply:
I restricted it to 2.3.4 because they introduced a patch fix for the `protobuf` version issues in that release (if you try to install an earlier version like 2.3.3, you get an incompatible version of `protobuf`).

Reply:
Makes sense, we could include that in `CHANGES.rst` as well (that 2.3.4 is the minimum supported) but no biggies either way.